src/condition-rule.ts

```diff
@@ -40,7 +40,7 @@ export class ConditionRule extends Rule {
 		this.conditionType = options.conditionType ? (typeof options.conditionType === "string" ? 
 			ConditionType.get(options.conditionType) : 
 			options.conditionType) :
-			Rule$ensureConditionType(options.name, rootType, options.category || "ErrorConditionType");
+			Rule$ensureConditionType(this, options.name, rootType, options.category || "ErrorConditionType");		
```

Comment:
This is not due to your changes, but one of the bugs I identified when planning this was that "ErrorConditionType" should be "Error". Do you mind fixing that while you're in here?

Reply:
fixed!